src/controllers/contactController.js

```diff
@@ -1,33 +1,34 @@
-const Contact = require('../models/contactModel')
 const contactService = require('../services/contactService')
+const { formatResponse, contactMessageTemplate: message } = require('../helpers/responseFormatter')
 
 const getContact = async (req, res) => {
-  let contactList = await Contact.find()
-  // if query params exist, search by name
-  // make middleware for query params
-  res.json({ message: "get all contacts", content: contactList })
+  // get data for database query
+  let { limit, page, name } = req.query
+  // (WIP) make middleware for query params
+  let contactList = await contactService.list(limit, page, name)
+  res.send(formatResponse(contactList))
```

Comment:
check usage of res.send vs res.end

Reply:
res.end mosty used if we dont want to send any response data. for most cases, res.send will do the job